src/utils.py

```diff
@@ -166,3 +181,12 @@ def read_view_content(view: Literal["block", "flow", "collection"]) -> Dict[str,
     resp = httpx.get(repo_url + f"/views/{view_filename}")
     resp.raise_for_status()
     return resp.json()
+
+
+@sync_compatible
+async def get_repo(name: str) -> github3.repos.repo.Repository:
+    """Returns a GitHub repository object for a given collection name."""
+
+    github_token = await Secret.load("collection-registry-github-token")
+    gh = github3.login(token=github_token.get())
```

Comment:
yeah that might be a good idea

Reply:
I think I will wrap all the github methods in `run_sync_in_worker_thread` to avoid scope creep here

the flow executes quite quickly as is, so this feels like a minor tweak